test/unit/rules/no-passed-in-event-handlers-test.js

```diff
@@ -6,7 +6,7 @@ const generateRuleTests = require('../../helpers/rule-test-harness');
 generateRuleTests({
   name: 'no-passed-in-event-handlers',
 
-  config: 'true',
```

Comment:
O_o

Reply:
Ya, I have no idea why this was stringified...